client/components/fields/DayPickerInput/index.jsx

```diff
@@ -1,6 +1,6 @@
 import React, { PropTypes } from 'react'
 import DatePicker from 'react-datepicker'
-import TimePicker from 'rc-time-picker'
+import { TimePicker } from '../TimePicker'
 import moment from 'moment'
 import 'react-datepicker/dist/react-datepicker.css'
 import 'rc-time-picker/assets/index.css'
```

Comment:
this can be removed too, like the ref to rc-time-picker in the packages.json file